proposals/lip-0007.md

```diff
@@ -6,16 +6,17 @@ Discussions-To: https://research.lisk.io/t/use-a-consistent-and-informative-vers
 Status: Active
 Type: Informational
 Created: 2018-09-14
-Updated: 2019-11-04
+Updated: 2020-06-22
 ```
 
 ## Abstract
 
-This LIP proposes a versioning system for Lisk Core aimed at better reflecting the ongoing changes to the Lisk software implementation and its underlying blockchain protocol. One which makes the clear distinction between software implementation and blockchain protocol changes.
+This LIP proposes a versioning system for Lisk Core, Lisk SDK and any white label blockchain application created using Lisk SDK.
+One which makes the clear distinction between software implementation and blockchain protocol changes.
 
-The changes to the Lisk software implementation are reflected through a semantic versioning scheme (https://semver.org/), relying on a 3 digit notation **(MAJOR.MINOR.PATCH)** to indicate major, minor and patch level changes.
+The software implementation changes to the Lisk Core and Lisk SDK are reflected through a semantic versioning scheme (https://semver.org/), relying on a 3 digit notation **(MAJOR.MINOR.PATCH)** to indicate major, minor and patch level changes.
 
-The changes to the Lisk blockchain protocol are additionally reflected using a 2 digit notation **(H.S)** to indicate chain and network compatibility, and whether the release will introduce a hard or soft fork event.
+The changes to Lisk protocol are additionally reflected using a 2 digit notation **(H.S)** to indicate chain and network compatibility, and whether the release will introduce a hard or soft fork event.
```

Comment:
```suggestion
For a specific blockchain network, its protocol version is represented using a 2 digit notation **(H.S)** to indicate chain and network compatibility, and whether the release will introduce a hard or soft fork event.
```

Reply:
I think the important think here is that we don't version the Lisk protocol, but the specific network.